applications/nrf_desktop/README.rst

```diff
@@ -1512,7 +1512,7 @@ To enable the MCUboot bootloader, select the :kconfig:`CONFIG_BOOTLOADER_MCUBOOT
 
 Configure the MCUboot bootloader with the following options:
 
-* ``CONFIG_BOOT_SIGNATURE_KEY_FILE`` - This option defines the path to the private key that is used to sign the application and that is used by the bootloader to verify the application signature.
+* :kconfig:`CONFIG_BOOT_SIGNATURE_KEY_FILE` - This option defines the path to the private key that is used to sign the application and that is used by the bootloader to verify the application signature.
```

Comment:
Please see lines 1616 - 1617 in this file. The `CONFIG`s are not written with :kconfig: before it. Should this be added?

Reply:
@MarekPieta , are these Kconfigs brought in into the NCS Kconfig system or they're still separate and specific to nRF Desktop?

Reply:
The `CONFIG_BOOT_SIGNATURE_KEY_FILE` comes from MCUboot and it's not an application-specific Kconfig. I am not sure if we already can link to MCUboot-specific Kconfig options in documentation, if I recall correctly that used not to be possible.

btw. We can already link nRF Desktop specific Kconfig options in documentation using `:kconfig:` syntax (there used to be an issue related to that, but it was fixed some time ago).

Reply:
I can see in the **Applications/nRF Desktop/README** file that there are some with the `:kconfig` syntax, but the `CONFIG_DESKTOP_` ones do not link to anything.

Should those with the `:kconfig:` in nRF Desktop link to [here](http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-6345/nrf/applications/nrf_desktop/README.html#cmdoption-arg-CONFIG_DESKTOP_INIT_LOG_BATTERY_STATE_EVENT)?

Reply:
Discussed offline

Reply:
Have removed the `:kconfig` syntax for the nRF Desktop specific options for now, as they are not global Kconfig options. Will bring this up in future meetings for further discussion.